src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -1747,10 +1767,10 @@ public static InetAddressAndPort getReplaceAddress()
     {
         try
         {
-            if (System.getProperty(Config.PROPERTY_PREFIX + "replace_address", null) != null)
-                return InetAddressAndPort.getByName(System.getProperty(Config.PROPERTY_PREFIX + "replace_address", null));
-            else if (System.getProperty(Config.PROPERTY_PREFIX + "replace_address_first_boot", null) != null)
-                return InetAddressAndPort.getByName(System.getProperty(Config.PROPERTY_PREFIX + "replace_address_first_boot", null));
+            if (REPLACE_ADDRESS.getString(null) != null)
```

Comment:
is not `getString(null)` redundant here? The same behavior is achieved with `getString()` as `null` is default anyway. Another option is to do `REPLACE_ADDRESS("cassandra.replace_address", null)` but why would one do it, right?

Reply:
I was done it intentionally to simplify the review. The whole PR is quite big, so even small refactoring may increase total complexity. 

I totally agree here. 
Fixed.